build.gradle.kts

```diff
@@ -1,17 +1,17 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    id("org.jetbrains.intellij") version "1.7.0"
-    kotlin("jvm") version "1.7.10"
-    id("org.jmailen.kotlinter") version "3.11.1"
+    id("org.jetbrains.intellij") version "1.10.0"
+    kotlin("jvm") version "1.7.21"
```

Comment:
Did you make sure this matches the version used by IDEA? We don't always use the latest version.

Reply:
It should be

Reply:
idk how to check tbh

Reply:
When you import the project in idea, there will be a kotlin dependency for idea itself listed in the libraries